CHANGELOG.md

```diff
@@ -1,3 +1,130 @@
+- Add an endpoint to list Vsphere datastores: GET /api/v2/providers/vsphere/datastores [#6442](https://github.com/kubermatic/kubermatic/issues/6442) 
+- KKP Installer will update Helm releases automatically if the values have changed (no need for `--force` in most cases). [#6449](https://github.com/kubermatic/kubermatic/issues/6449) 
+- Introduce resource quota for Alibaba provider [#6458](https://github.com/kubermatic/kubermatic/issues/6458) 
+- Add Gatekeeper health to cluster health status [#6461](https://github.com/kubermatic/kubermatic/issues/6461) 
+- Remove CoreOS Support [#6465](https://github.com/kubermatic/kubermatic/issues/6465) 
+- Relax memory limit for openvpn container [#6467](https://github.com/kubermatic/kubermatic/issues/6467) 
+- Enable VM resource quota for AWS provider [#6468](https://github.com/kubermatic/kubermatic/issues/6468) 
+- Multus has been added as an addon.  [#6477](https://github.com/kubermatic/kubermatic/issues/6477) 
+- CoreDNS version now based on Kubernetes version [#6501](https://github.com/kubermatic/kubermatic/issues/6501) 
+- Add support for "use-octavia" setting in Openstack provider specs. It defaults to "true" but leaves the possibility to set it to "false" if your provider doesn't support Octavia yet but Neutron LBaaSv2
+ [#6529](https://github.com/kubermatic/kubermatic/issues/6529) 
+- Add components override field to set nodeportrange for apiserver [#6533](https://github.com/kubermatic/kubermatic/issues/6533) 
+- Extend Kubermatic Constraint Template spec with a selector which enables cluster filtering for EE Kubermatic.  [#6534](https://github.com/kubermatic/kubermatic/issues/6534) 
+- Overhaul CA handling, allow to configure a global CA bundle for every component. The OIDC CA ile has been removed, manual updates can be necessary. [#6538](https://github.com/kubermatic/kubermatic/issues/6538) 
+- OpenShift support is removed.  [#6539](https://github.com/kubermatic/kubermatic/issues/6539) 
+- Fix issue with gatekeeper not recognizing the AdmissionReview v1 version by changing the webhook to use v1beta1 [#6550](https://github.com/kubermatic/kubermatic/issues/6550) 
+- OpenStack: Add support for "use-octavia" setting in Cluster Openstack cloud specs [#6565](https://github.com/kubermatic/kubermatic/issues/6565) 
+- Kubermatic Constraint Templates now support cluster filtering to allow users to manage the CT's and policies across different clusters more easily. The feature is available for Kubermatic EE.  [#6567](https://github.com/kubermatic/kubermatic/issues/6567) 
+- Fix KAS service port in Tunneling agent configuration. [#6569](https://github.com/kubermatic/kubermatic/issues/6569) 
+- Add support for Hetzner CCM [#6588](https://github.com/kubermatic/kubermatic/issues/6588) 
+- Increase default gatekeeper webhook timeout from 2 to 10 seconds, and added option in cluster settings to configure it. [#6590](https://github.com/kubermatic/kubermatic/issues/6590) 
```

Comment:
There was another change to the default timeout, putting it to 3 s. So you can merge this with:

`Change default gatekeeper webhook timeout to 3 sec [#6709](https://github.com/kubermatic/kubermatic/issues/6709) ` 

and put `...from 2 to 3 seconds..`

Reply:
Okay